scripts/ssl-manager.js

```diff
@@ -868,6 +928,8 @@ function SSLManager(config) {
 
     //managing certificate challenge validation by routing all requests to master node with let's encrypt engine
     me.manageDnat = function manageDnat(action) {
+        if (config.webroot) return { result: 0 }
```

Comment:
```suggestion
        if (config.webroot) return { result: 0 };
```

Reply:
avoid confusion and don't call manageDnat at all (because it will appear in logs)